src/app/auth/sign-up/page.tsx

```
@@ -1,8 +1,14 @@
+import { auth } from "@/lib/auth";
 import { SignUpViews } from "@/modules/auth/ui/views/sign-up-views";
+import { headers } from "next/headers";
+import { redirect } from "next/navigation";
 import React from "react";
 
-const Page = () => {
+const Page = async() => {
+  const session = await auth.api.getSession({
+        headers: await headers()
+      });
+      if(!!session)redirect("/");
```

Biome (1.9.4)

[error] Avoid redundant double-negation.

It is not necessary to use double-negation when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant double-negation


(lint/complexity/noExtraBooleanCast)